tests/rp_cloud_cleanup.py

```diff
@@ -437,6 +468,75 @@ def get_net_id_from_nat(nat):
                         'network-'):
                     return tag['Value'].split('-')[1]
 
+        def get_date_from_resource(aws_res):
+            if 'CreateTime' in aws_res:
+                return datetime.strftime(aws_res['CreateTime'],
+                                         "%Y-%m-%d %H:%M:%S")
+            else:
+                return "Unknown"
+
+        def check_cloud_network(aws_res):
+            # This resource is created by CloudV2 API
+            _net_id = get_net_id_from_nat(aws_res)
```

Comment:
In general I'm not not following the use of `_`-prefixed local variables here and in other changed. "In general" (in my impression) and specifically in the rptest codebase we use `_`-prefix for private _instance_ variables. Locals (which by their nature don't have a public/private distinction) are unprefixed.

Reply:
I am using this for code visibility and to right away understand that this variable is local. It is a habit from my C-lang days. And yes, it is pretty useless in Python since all vars are accessible.
Will update.